source/operations/concepts/architecture.rst

```diff
@@ -0,0 +1,189 @@
+      Adding new peers uses the same sequence for synchronizing data.
+
+Replication performance primarily depends on the network latency between each peer site.
+   With geographically distributed peer sites, high latency between sites can result in significant replication lag.
+   This can compound with workloads that are near or at the deployment's overall performance capacity, as the replication process itself requires 
+
+   .. figure:: /images/architecture/architecture-multi-site-latency.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with latency between sites
+
+      In this peer configuration, the latency between Site A and it's peer sites is 100ms.
+      The earliest point in time in which the object fully synchronizes to all sites is at least 110ms.
+
+Deploying a global load balancer or similar network appliance with support for site-to-site failover protocols is critical to the functionality of multi-site deployments.
+   The load balancer should support a health probe/check setting to detect the failure of one site and automatically redirect applications to any remaining healthy peer.
+
+   .. figure:: /images/architecture/architecture-load-balancer-multi-site.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with a failed site
+
+      One of the peer sites has failed completely.
+      The load balancer automatically routes requests to the remaining healthy peer site.
+
+   The load balancer should meet the same requirements as single-site deployments regarding connection balancing and header preservation.
+   Once all data synchronizes, you can restore normal connectivity to that site.
+   Depending on the amount of replication lag, latency between sites, and overall workload IO, you may need to temporarily stop write operations to allow the sites to completely catch up.tions to other sites in the deployment.
+
+MinIO replication can automatically heal a site that has partial data loss due to transient or sustained downtime. 
+   If a peer site completely fails, you can remove that site from the configuration entirely.
+   The load balancer configuration should also remove that site to avoid routing client requests to the offline site.
+
+   You can then restore the peer site, either after repairing the original hardware or replacing it entirely, by adding it back to the site replication configuration.
+   MinIO automatically begins resynchronizing content.
+
+   .. figure:: /images/architecture/architecture-load-balancer-multi-site-healing.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with a healing site
+
+      The peer site has recovered and reestablished connectivity with its healthy peers.
+      MinIO automatically works through the replication queue to catch the site back up.
+
+   Once all data synchronizes, you can restore normal connectivity to that site.
```

Comment:
Should this and following be identical text to 171-172?

Reply:
Whoops. Nope.